### PR TITLE
ft: ZENKO-18: Metadata Proxy Server

### PR DESCRIPTION
**Pull request by ploki**

### CODE REVIEW

**Review by ssalaues — DISMISSED** [review]

**Review by vrancurel — DISMISSED** [review]

File: lib/storage/metadata/proxy/BucketdRoutes.js [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+              this._logger.newRequestLogger() :
+              this._logger.newRequestLoggerFromSerializedUids(uids);
+        logger.trace('new request', { method: req.method, url: req.url });
+        return logger;
+    }
+
+    // Internal routes
+
+    /**
+     * Handle routes related to operations on buckets attributes
```
Comment (jonathan-gramain) [inline lib/storage/metadata/proxy/BucketdRoutes.js]: on bucket~s~ attributes

File: lib/storage/metadata/proxy/BucketdRoutes.js [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+        const adminNamespace = '_';
+        const logger = this._createRequestLogger(req);
+        const uriComponents = getURIComponents(req.url, logger);
+        if (!uriComponents) {
+            return sendResponse(req, res, logger, errors.BadRequest);
+        }
+        switch (uriComponents.namespace) {
+        case adminNamespace:
+            return this._adminRoutes(req, res, uriComponents, logger);
+        default: // coincidently matches the `default' literal namespace as well
```
Comment (jonathan-gramain) [inline lib/storage/metadata/proxy/BucketdRoutes.js:311]: Is anything starting with something else than `/_/` a default route? (so `/default/...` but also `/foobar/...`) It's ok for me just wondering.

**Review by jonathan-gramain — DISMISSED** [review]

File: lib/storage/metadata/proxy/BucketdRoutes.js (hunk 2 above)
Comment (ploki) [reply lib/storage/metadata/proxy/BucketdRoutes.js:311]: Yes, that's it

**Comment by ploki** [comment]
replaced by #498 to please @bert-e on the branch name convention

**Pull request closed without merging**